.github/workflows/insta-merge.yaml

```diff
@@ -0,0 +1,28 @@
+name: Insta Merge Konflux PRs
```

Comment:
@coderabbitai full

Reply:
@coderabbitai also, the workflow breaks our yaml linter, please create github issue to address